python/pyspark/mllib/clustering.py

```diff
@@ -466,7 +557,8 @@ def predictOnValues(self, dstream):
 
 def _test():
     import doctest
-    globs = globals().copy()
+    import pyspark.mllib.clustering
+    globs = pyspark.mllib.clustering.__dict__.copy()
```

Comment:
I don't remember why this is required, but thanks for the explanation and let's keep it this way.